src/app/playlists/[id]/swipe/page.tsx

```
@@ -0,0 +1,61 @@
+      </div>
+    );
+  if (tracks.length === 0)
+    return (
+      <div className="min-h-screen bg-black text-white flex flex-col items-center justify-center gap-4">
+        <p>曲がありません</p>
+        <button
+          onClick={() => router.push(`/playlists/${id}`)}
+          className="text-green-400"
+        >
+          プレイリストに戻る
+        </button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)